app/src/gui/fields/TakePhoto.tsx

```diff
@@ -15,207 +15,292 @@
+        textAlign: 'center',
+        bgcolor: theme.palette.grey[100],
+        borderRadius: theme.spacing(2),
+        marginTop: theme.spacing(2),
+      }}
+    >
+      <CameraAltIcon sx={{fontSize: 48, color: 'text.secondary', mb: 2}} />
+      <Typography variant="h6" gutterBottom>
+        No Photos Yet
+      </Typography>
+      <Button
+        variant="contained"
+        color="primary"
+        onClick={onAddPhoto}
+        startIcon={<CameraAltIcon />}
+      >
+        Take First Photo
+      </Button>
+    </Paper>
   );
 };
 
-const FAIMSImageIconList = (props: {
-  index: number;
-  setopen: Function;
-  fieldName: string;
-}) => {
-  const {index, setopen} = props;
+/**
+ * Displays a placeholder when an image is unavailable or cannot be loaded
+ */
+const UnavailableImage = () => {
+  const theme = useTheme();
   return (
-    <ImageListItem key={`${props.fieldName}-image-icon-${index}`}>
-      <IconButton aria-label="image" onClick={() => setopen(null)}>
-        <ImageIcon />
-      </IconButton>
-    </ImageListItem>
+    <Box
+      sx={{
+        width: '100%',
+        height: '100%',
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        backgroundColor: theme.palette.background.lightBackground,
+      }}
+    >
+      <ImageIcon sx={{fontSize: 48, color: 'text.secondary'}} />
+    </Box>
   );
 };
 
-const FAIMSImageList = (props: ImageListProps) => {
-  const {images, setopen, setimage, fieldName} = props;
-  const disabled = props.disabled ?? false;
-  const handelonClick = (index: number) => {
+/**
+ * Displays a grid of images with add and delete functionality
+ */
+const ImageGallery = ({
+  images,
+  setOpen: setopen,
+  setImages,
+  disabled,
+  fieldName,
+  onAddPhoto,
+}: ImageListProps) => {
+  const theme = useTheme();
+  // Handler for deleting images from the gallery
+  const handleDelete = (index: number) => {
     if (images.length > index) {
-      const newimages = images.filter((image: any, i: number) => i !== index);
-      setimage(newimages);
+      // need to reverse the index here to account for reverse display
+      const originalIndex = images.length - 1 - index;
+      const newImages = images.filter(
+        (_: any, i: number) => i !== originalIndex
+      );
+      setImages(newImages);
     }
   };
 
-  if (images === null || images === undefined)
-    return <span>No photo taken.</span>;
-  if (disabled === true)
-    return (
-      <FAIMSViewImageList
-        images={props.images}
-        fieldName={props.fieldName}
-        setopen={setopen}
-      />
-    );
   return (
-    <ImageGalleryList>
-      {props.images.map((image: any, index: number) =>
-        image['attachment_id'] === undefined ? (
-          <ImageListItem key={`${fieldName}-image-${index}`}>
-            <img
-              style={{
-                objectFit: 'scale-down',
-                cursor: 'allowed',
+    <Box sx={{width: '100%'}}>
+      <Box
```

Comment:
I feel like the thumbnail images are too big, they take up the whole display width on mobile. Maybe try to fit 2-3 images across on mobile? Need to be able to see the general image but not detail - at the moment the thumbnail is as big as the popup image.

Reply:
Done